js/main.js

```diff
@@ -21,6 +26,20 @@ function Main(){
   this.isStarted = false;
 }
 
+//Function for stopping and starting audio playback -Z
+function MusicSwitch(){
+if(sw){
```

Comment:
@zseiman instead of using a boolean variable `sw` to check if the music is playing, you can use the property `paused` of a HTML audio tag. So in this case, `if (sw){` will be `if (!audio.paused){`.